server/common/admin-rest-client/src/main/java/org/kaaproject/kaa/server/common/admin/AdminClient.java

```diff
@@ -1062,26 +1062,25 @@ public UserDto editUserProfile(UserDto userDto) {
         return response.getBody();
     }
 
-    public CredentialsDto provisionCredentials (String applicationId, String credentials) {
+    public CredentialsDto provideCredentials (String applicationId, String credentialsBody) {
```

Comment:
According to the [Google Java Style](https://google.github.io/styleguide/javaguide.html#s5-naming), a method name should be either a verb or a verb phrase and reflect what the method actually does. The method we are talking about provides security credentials given, hence the name.